frontend/src/modules/studySamples/services.ts

```diff
@@ -109,8 +109,9 @@ export async function fetchSamplesAndLibraries(sampleList: number[]) {
 	return availableSamples
 }
 
-export async function fetchCompletedSamples(studyID: FMSId, stepOrderID: FMSId, workflow_action: WorkflowActionType, limit: number, offset: number) {
-	const stepHistoryResponse = await store.dispatch(api.stepHistory.getCompletedSamplesForStudy(studyID, {step_order__id__in: stepOrderID, limit, offset, workflow_action}))
+export async function fetchCompletedSamples(studyID: FMSId, stepOrderID: FMSId, workflow_actions: WorkflowActionType[], limit: number, offset: number) {
+  const qsWorkflowActions = workflow_actions.join(" ")
+	const stepHistoryResponse = await store.dispatch(api.stepHistory.getCompletedSamplesForStudy(studyID, {step_order__id__in: stepOrderID, limit, offset, workflow_action: qsWorkflowActions}))
```

Comment:
If you want to be extra cool, you can make the type more precise for getCompletedSamplesForStudy ;)
```ts
    getCompletedSamplesForStudy: (studyId, options) => get<JsonResponse<FMSPagedResultsReponse<any>>>('/step-histories/', {...options, study__id__in: studyId}),
```

Reply:
nope i am not a cool type guy